Sources/Bow/Data/Either.swift

```diff
@@ -181,6 +181,18 @@ public extension Either {
     public static func eq<EqL, EqR>(_ eql : EqL, _ eqr : EqR) -> EitherEq<A, B, EqL, EqR> {
         return EitherEq<A, B, EqL, EqR>(eql, eqr)
     }
+
+    public static func bifunctor() -> EitherBiFunctor<A, B> {
+        return EitherBiFunctor<A, B>()
+    }
+}
+
+public class EitherBiFunctor<A, B>: Bifunctor {
```

Comment:
Bifunctor is usually written without uppercase in the `f`. Like this:
```suggestion
public class EitherBifunctor<A, B>: Bifunctor {
```